src/gap.c

```diff
@@ -535,19 +535,18 @@ Obj FuncRETURN_NOTHING (
 Obj FuncRuntime (
                  Obj                 self )
 {
-  return INTOBJ_INT( SyTime() );
+    return ObjInt_UInt(SyTime());
 }
 
 
 Obj FuncRUNTIMES( Obj     self)
 {
   Obj    res;
   res = NEW_PLIST(T_PLIST, 4);
-  SET_LEN_PLIST(res, 4);
-  SET_ELM_PLIST(res, 1, INTOBJ_INT( SyTime() ));
-  SET_ELM_PLIST(res, 2, INTOBJ_INT( SyTimeSys() ));
-  SET_ELM_PLIST(res, 3, INTOBJ_INT( SyTimeChildren() ));
-  SET_ELM_PLIST(res, 4, INTOBJ_INT( SyTimeChildrenSys() ));
+  PushPlist(res, ObjInt_UInt(SyTime()));
+  PushPlist(res, ObjInt_UInt(SyTimeSys()));
+  PushPlist(res, ObjInt_UInt(SyTimeChildren()));
+  PushPlist(res, ObjInt_UInt(SyTimeChildrenSys()));
```

Comment:
Note that the code using `RUNTIMES` relies on the various parts being in a specific position each. So I kind of like the explicit indices, as it makes it easy to compare this code with its high-level wrapper in the GAP library and verify that they match. Perhaps if this code was changed to simply use `ASS_LIST` instead of `SET_ELM_PLIST`? That should be as safe or "safer" than `PushPlist`, but retain the explicit enumeration.

Of course it's only four entries here, so it's not a bridge I am going to die on, but I feel that it needlessly increases the cognitive load here.

Reply:
I'm happy with using ASS_LIST instead